uportal-war/src/main/webapp/media/skins/respondr/common/less/navigation.less

```diff
@@ -43,6 +45,47 @@
         }
     }
 
+   .logout-launcher,.notification-icon,.portal-greeting,.login-launcher{
+        display: block;
+        float: right;
+        margin-right: 5px;
+        margin-top: 12px;
+        word-break: keep-all;
+    }
+
+    #portalCASLogin{
+        display: block;
+        float: right;
+        margin-right: 5px;
+        margin-top: -5px;
+    }
```

Comment:
No it isn't specific to our implementation
https://github.com/Jasig/uPortal/blob/1c88afbdd7370f932ec1891d4e4ba039660ae851/uportal-war/src/main/webapp/WEB-INF/jsp/Invoker/login.jsp
:-)